WeakAuras/Profiling.lua

```diff
@@ -316,14 +316,32 @@ local function PrintOneProfile(popup, name, map, total)
   if map.count ~= 0 then
     popup:AddText(name .. "  ERROR: count is not zero:" .. " " .. map.count)
   end
+
   local percent = ""
   if total then
     percent = ", " .. string.format("%.2f", 100 * map.elapsed / total) .. "%"
   end
+
   local spikeInfo = ""
+  
   if map.spike then
-    spikeInfo = string.format("(%.2fms)", map.spike)
+    local color
+    local r, g, b, a
+
+    if map.spike < 2 then
+      r, g, b, a = WeakAuras.GetHSVTransition(map.spike / 2, 0, 1, 0, 1, 1, 1, 0, 1)
+    elseif map.spike >= 2 and map.spike < 3 then
+      r, g, b, a = WeakAuras.GetHSVTransition((map.spike - 2) , 1, 1, 0, 1, 1, 0.65, 0, 1)
+    else 
+      r, g, b, a = WeakAuras.GetHSVTransition(1, 1, 0, 0, 1, 1, 0, 0, 1)
```

Comment:
Why would you interpolate here between two colors?

Reply:
For the else part indeed no point

Reply:
Now that I think about it, I probably need a transition from orange to red as well. Instead of just orange at 2.99, followed by red at 3